kafka-ui-react-app/src/components/common/DiffViewer/__tests__/DiffViewer.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import { shallow } from 'enzyme';
```

Comment:
Do we really need to have `enzyme` in new code when we have issue to get rid of it?